staging/src/k8s.io/apiserver/pkg/server/options/tracing.go

```diff
@@ -23,9 +23,9 @@ import (
 	"net"
 
 	"github.com/spf13/pflag"
-	"go.opentelemetry.io/otel/exporters/otlp/otlpgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
 	"go.opentelemetry.io/otel/sdk/resource"
-	"go.opentelemetry.io/otel/semconv"
+	"go.opentelemetry.io/otel/semconv/v1.12.0"
```

Comment:
@dashpole are there any user-facing impacts of this version change we need to release note?

Reply:
This is not user-facing.  The conventions we use (just service.name) do not differ between versions.